torch/testing/_internal/common_optimizers.py

```diff
@@ -1255,6 +1255,21 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
                 "TestOptimRenewed",
                 "test_fused_matches_forloop",
             ),
+            DecorateInfo(
+                # Note on tolerances:
+                # test_grad_scaling_autocast_fused_optimizers_Adam_cuda_float32
+                # Mismatched elements: 64 / 64 (100.0%)
+                # Greatest absolute difference: 2.3484230041503906e-05 at index (7, 5) (up to 1e-05 allowed)
+                # Greatest relative difference: 3.567120074876584e-05 at index (5, 5) (up to 1.3e-06 allowed
```

Comment:
well, we don’t know why this is the case. could you simplify this comment to just link to the issue tracking the discrepancy?

same for below